main.go

```diff
@@ -130,17 +125,21 @@ func main() {
 
 	}
 
-	sender := metric_sender.NewMetricSender(dropsonde.AutowiredEmitter())
+	dropReporter := initializeDropsondeReporter(prefix, logger, c)
+	promReporter := initializePrometheusReporter(c)
+	registryMetrics := metrics.NewMultiRouteRegistryReporter(dropReporter, promReporter)
+	proxyMetrics := metrics.NewMultiProxyReporter(dropReporter, promReporter)
+	monitorMetrics := metrics.NewMultiMonitorReporter(dropReporter, promReporter)
```

Comment:
Please make sure that the list of reporters for the various `*Multi*Reporter`s just doesn't contain nil values by passing a list, not 2 args. Should make it simpler and less error prone.